gprofiler/main.py

```diff
@@ -557,6 +490,28 @@ def parse_cmd_args():
     return args
 
 
+def _add_profilers_arguments(parser):
+    profilers_registry = get_profilers_registry()
+    for profiler_name, profiler_config in profilers_registry.items():
```

Comment:
Please, shorter names. All names here are very long. `profiler_name` can be `name`. `profilers_registry` can be `registry`. `profiler_argument_group` can be just `arg_group`. I think it's obvious enough from context.